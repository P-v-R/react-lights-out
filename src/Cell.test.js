import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Cell from "./Cell";

test("renders without breaking", function () {
  render(
    <table>
      <tbody>
        <tr>
          <Cell isLit={true} />
        </tr>
      </tbody>
    </table>
  );
});

test("watches snapshot", function() {
  const {container} =   
  render(
    <table>
      <tbody>
        <tr>
          <Cell isLit={true} />
        </tr>
      </tbody>
    </table>
  );
  expect(container).toMatchSnapshot();
});