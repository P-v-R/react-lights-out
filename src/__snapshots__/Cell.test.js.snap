// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`watches snapshot 1`] = `
<div>
  <table>
    <tbody>
      <tr>
        <td
          class="Cell Cell-lit"
        />
      </tr>
    </tbody>
  </table>
</div>
`;
